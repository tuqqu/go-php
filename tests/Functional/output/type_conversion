1 1 2 57.4 57
97 241 1105 101 239 340 hello world
