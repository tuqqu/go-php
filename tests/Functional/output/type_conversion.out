1 1 2.000000 57.400000 57
97 250 1105 101 239 340 hello world heïŔ ú a
